draft-ietf-quic-recovery.md

```diff
@@ -848,8 +848,8 @@ Pseudocode for SetLossDetectionTimer follows:
       return
     if (loss_time != 0):
       // Early retransmit timer or time loss detection.
-      timeout = loss_time -
-        time_of_last_sent_retransmittable_packet
+      loss_detection_timer.set(loss_time)
+      return
     else:
```

Comment:
You don't need an else clause if you are going to exit.  But this might set the timer to some time in the past if we're not careful with setting loss_time.  I have a PR that might help inbound.